docs/faq.md

```diff
@@ -1,10 +1,22 @@
 # FAQ
 
-## How can I contact the developers?
+## What is PMM?
 
-- [Community forum](https://www.percona.com/forums/questions-discussions/percona-monitoring-and-management).
-- [Discord chat](http://per.co.na/discord).
-- [PMM project in JIRA](https://jira.percona.com/projects/PMM).
+Percona Monitoring and Management (PMM) is an open-source tool for monitoring databases and the servers they run on.
+
+PMM extracts metrics data from your databases and charts them in graphical and tabular form.
+
+## What can it do for me?
+
+- Track performance over time
+- Identify problem queries
+- Identify latencies and resource bottlenecks
+
+## How much does it cost?
+
+PMM is free if you run PMM on your own hardware.
+
+If you run on a cloud service (Amazon AWS) there are infrastructure fees.
```

Comment:
there may be fees...only if you deploy from the marketplace...  your ec2 instances with the pmm binaries are still free